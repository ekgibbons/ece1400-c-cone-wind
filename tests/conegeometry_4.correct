Usage:
    $ ./conegeometry <radius> <height> <area_or_volume>

    area_or_volume:  must be area or volume
