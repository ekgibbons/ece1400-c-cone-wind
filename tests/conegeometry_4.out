Usage:
    $ ./conegeometry <height> <radius> <area_or_volume>

    area_or_volume:  must be area or volume
